languages/ruby/exercises/concept/ostruct/.meta/design.md

```diff
@@ -0,0 +1,15 @@
+## Learning objectives
+
+- Understand how to require from stdlib
+- Understand OpenStruct
+- Learn &: block syntax
+
+## Concepts
+
+- `require`: Know how to use require
+- `ostruct`: Know how to use OpenStruct
```

Comment:
Is the `&:` construct also a separate concept?

Reply:
Kind of what I am thinking, as the this class doesn't provide it directly, it is available if we construct our code in such a way that we can use it, i.e., by defining a method.